pynamodb/models.py

```diff
@@ -349,12 +349,12 @@ def update_item(self, attribute, value=None, action=None, conditional_operator=N
         if len(expected_values):
             kwargs.update(expected=self._build_expected_values(expected_values, UPDATE_FILTER_OPERATOR_MAP))
         kwargs[pythonic(ATTR_UPDATES)] = {
-            attribute: {
+            attribute_cls.attr_name: {
```

Comment:
i wonder if we should pull the name out up [here](https://github.com/jlafon/PynamoDB/pull/160/files#diff-66d6c6d3d504b9e11a4e236e67133ddeR343) and save to a local var